app/assets/stylesheets/components/modals.scss

```diff
@@ -1,32 +1,80 @@
 @import '../config/import';
 
-%modal-style {
-  display: inline-block;
-  width: 100%;
-}
-
 .crayons-modal {
-  @extend %modal-style;
-  @include generate-box(
-    $level: 3,
-    $bg: var(--card-bg),
-    $border: var(--card-color),
-    $color: var(--card-color)
-  );
-  padding: $su-7;
-  max-width: 640px;
+  position: fixed;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
```suggestion
  place-items: center;
```

Reply:
woooo, smart! i love it! <3

Reply:
oh wait.. it doesn't work.. it does `justify-items` instead of `justify-content`. This would work if that was a `grid` instead of a `flex`. But grid won't behave the same way, so I'll stick to original solution.